compiler/native/expand_test.go

```diff
@@ -538,7 +539,7 @@ func TestNative_ExpandStepsMulti(t *testing.T) {
 					"auth_method": "token",
 					"username":    "octocat",
 					"items": []interface{}{
-						map[interface{}]interface{}{string("path"): string("docker"), string("source"): string("secret/docker")},
+						map[interface{}]interface{}{"path": "docker", "source": "secret/docker"},
```

Comment:
The `string` is redundant since is able to infer that anything within `""` is already a string.